pkgs/development/python-modules/qtile/default.nix

```diff
@@ -70,6 +71,7 @@ buildPythonPackage rec {
     dbus-next
     dbus-python
     iwlib
+    libcst
     mpd2
     psutil
     pulsectl-asyncio
```

Comment:
I think based on the code pytz is preferred over dateutil https://github.com/qtile/qtile/blob/e4b4bb8b6f2c07570b049e7e9171f5ce6da0625c/libqtile/widget/clock.py#L39